types/pg/index.d.ts

```diff
@@ -54,6 +54,11 @@ export interface QueryConfig<I extends any[] = any[]> {
     name?: string;
     text: string;
     values?: I;
+    types?: CustomTypesConfig;
+}
+
+export interface CustomTypesConfig {
+    getTypeParser: (id: pgTypes.TypeId, format: pgTypes.TypeFormat) => any;
```

Comment:
This type definition is out of date for the latest version of `pg-types`, but `pg` v7.14 uses `pg-types` ^v2.1.0: https://github.com/brianc/node-postgres/blob/30f67bb246cabfc6a9d4cd026e6abedbcd0e2239/package.json

The bundled TypeScript declarations were added in `pg-types` v2.2.0: [This tag](https://github.com/brianc/node-pg-types/tree/d9d9dfb87eb50914043cbc178c301b7dd3b0c50d) and here is the relevant line: https://github.com/brianc/node-pg-types/blob/d9d9dfb87eb50914043cbc178c301b7dd3b0c50d/index.d.ts#L135

I have simply taken the definitions from that point in time and applied them here.